### DISCUSSION TITLE
Will it be possible to have the file/image selector inside JSON repeater ?

### DISCUSSION THREAD

**Discussion by stouch** [directus/directus#9455] [Archive]
Hi,

I LOVE Directus. Even if there are still little bugs, it's the best Headless API+CMS I've ever used.

But there is ONE thing that is missing : 
I'd really appreciate to be able to use the Library File/Image selector inside a UUID field inside a JSON repeater.

Is something that will be possible in a later version ?

Thank you !

**Comment by benhaynes** [comment]
Merging this into: https://github.com/directus/directus/discussions/3001
